lib/library.rb

```diff
@@ -0,0 +1,18 @@
+require 'yaml'
+
+class Library
+
+  attr_accessor :books
+
+  def initialize(books)
```

Comment:
What the "hound" is saying is that `books` is never used. When you create a new `Library`, you'll have to give it a piece of information that is not used. So: `library = Library.new('my books')`
`@books` and `books` are *not* the same variable.